PWGLF/Tasks/Resonances/phianalysis.cxx

```diff
@@ -288,7 +300,7 @@ struct phianalysis {
   }
   PROCESS_SWITCH(phianalysis, processMCLight, "Process Event for MC", false);
 
-  void processMCTrue(aod::ResoMCParents& resoParents)
+  void processMCTrue(aod::ResoCollision& collision, aod::ResoMCParents& resoParents)
```

Comment:
`collision` is not used.

Reply:
Thanks for pointing out this! I'll apply it as suggested.

Reply:
I had to use the collision info... I forgot to update it.